README.md

```diff
@@ -21,12 +22,38 @@ in the course and create one group for each section and add all the users in eac
     * To download modules to default users home directory as part of Python system path just do `pip install --user -r requirements.txt`. 
 4. Add appropriate properties to the `config.yaml` and `security.yaml` as show in the template under `config/config.yaml`, `config/security.yaml` and could be placed out side of source code.
 5. Logging configurations are in `logging.yaml` the file can be present out side of the source code. The logs can be logged to console or to file. Use the template provided in the `config/logging.yaml` and add
-    appropriate path to log file. The log get appended to `log/cgs.log` by default. Recommended keep log file out side of the utility
-## Run Directions 
+    appropriate path to log file. The log get appended to `log/cgs.log` by default. 
+#### Run Directions 
 
 1. The `run.sh` is used for running the utility and is checked in to the source code. A default configuration is provided to get going once your are set up. You can run this utility as  `./run.sh`
 2. The process may take long time for many sections in a course so it is good idea to run it in the background so that it continues after logging out from a machine. So use below while running on Linux servers.
 
     `nohup ./run.sh > /path-to-file/logFile.log &` 
 
+### Running the script in OpenShift Locally
+
+1. Download the [OpenShift CDK](http://developers.redhat.com/products/cdk/get-started/#tab-macLinux) follow along the CDK installation guidelines. 
```

Comment:
Put the generic CDK instructions in the wiki.  They are useful to many people and won't get updated if they are in here.